### DISCUSSION TITLE
Move tab to another window?

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#762] [Ideas]
### What feature would you like to see?

Like FireFox, I'd like the ability to move a tab to another window via the context menu.  If there's more than one window open, I'd like to be able to choose which window to which to send the tab, like FireFox does.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/689_

**Comment by jimfr** [comment]
Currently, Zen only has a "Move Tab to New Window" context menu item.  It would be nice if, like Edge, Vivaldi, and others, Zen would allow you to move a Tab to an existing window (I do this all the time, as I use windows to organize my work.  Here is what Edge does with only one window open: 

<img width="324" alt="image" src="https://github.com/user-attachments/assets/c1c29f48-0124-4f06-ad2d-cfe238c6f968">

Here is what Edge does with more than one window open, it gives you a choice of which window to which to move the tab (including a new window):

<img width="792" alt="image" src="https://github.com/user-attachments/assets/1095a4a4-f21c-41f1-a119-6f86c1ef428b">

Firefox allows this as well, though I like the chromium interface to this better, as it's simpler.

Thanks for listening! :)

**Comment by jimfr** [comment]
I posted some different screenshots here: https://github.com/zen-browser/desktop/issues/1259

Any thoughts on this?  It's something I would like to do all the time.  Thank you.

**Comment by maclarel** [comment]
+1 for this. It's quite annoying having to manually open the sidebar (I keep it collapsed at all times) just to move a tab to a different window. When working across multiple workspaces (both virtual desktops and Zen workspaces) this is also a tremendous pain in the neck.

Barring this specific functionality, it would be great if the sidebar would at least fly-out (as with a normal mouseover) while a tab is being dragged so that it can be dropped in without the need to manually trigger sidebar visibility _before_ moving the tab.

**Comment by hypermanketonic** [comment]
pls pls pls, I also want this

**Comment by SheepDomination** [comment]
Why was this never in zen to begin with ?

**Comment by zplawecki** [comment]
+1 for this one, one last feature I miss from Arc to be honest

**Comment by ShawkMusic** [comment]
A keyboard shortcut for this would be very nice too

**Comment by vsheyanov** [comment]
Hello everyone, this is not a native way to achieve what you want, but can be useful if you're OK with `Moving Active Tab to a Next Window` using a keyboard shortcut.
https://github.com/zen-browser/desktop/discussions/11419

**Comment by SheepDomination** [comment]
Everything else but what should be suggested.
